### PR TITLE
Add table is sqllab view in export

### PR DESCRIPTION
**Pull request by smacker**
Fix: #152 

As discussed before by hiding SQL Lab tables.

### CODE REVIEW

File: srcd/dashboards/gitbase/overview.json [hunk 1]
```diff
@@ -518,6 +521,7 @@
                 "description": null,
                 "fetch_values_predicate": null,
                 "filter_select_enabled": false,
+                "is_sqllab_view": false,
```
Comment (carlosms) [inline srcd/dashboards/gitbase/overview.json]: why is this one the only false?

Comment (smacker) [reply srcd/dashboards/gitbase/overview.json]: because one of the charts uses default pre-registered table. Pre-registred tables should be visible.

**Comment by carlosms** [comment]
let's also add an entry to the changelog, please!

**Comment by smacker** [comment]
fixed

**Pull request merged**